fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -27,10 +27,23 @@ export default Ember.Route.extend({
       });
   },
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    //alert("startPolling in setupController");
+    controller.stopPolling();
```

Comment:
this is the key part here. I call `stopPolling` before `startPolling`. Otherwise, there will be an increasing number of loops started, each of which are 5 sec intervals.